tools/corroborate/README.md

```
@@ -0,0 +1,61 @@
+go run ./tools/corroborate -in <evidence-dir> -out <output-dir> [-allowlist <allowlist.yaml>]
+```
+
+| Flag | Default | Meaning |
+|------|---------|---------|
+| `-in` | _(required)_ | Root of the source-keyed evidence tree (the GCS layout synced to disk, e.g. via `gcloud storage rsync`). Every `meta.json` beneath it is read with its sibling `ctrf/<phase>.json` reports. |
+| `-out` | `dist/corroborate` | Output directory. |
+| `-allowlist` | _(none)_ | Optional signer allowlist (`recipes/evidence/allowlist.yaml`). When set, each source's class is **re-derived** from its verified `(issuer, identity)` instead of trusting the `signer.class` baked into `meta.json`. |
+
+### Output layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)